README.md

```diff
@@ -3,7 +3,7 @@
 
 This is a Babel plugin that annotates React components with stable attributes that can be used to search and select using [FullStory](https://www.fullstory.com/). This is most useful when using a React system that generates dynamic names for Components or rearranges elements.
 
-For React on the web the attributes are `data-component`, `data-element`, and `data-source-file`. For React Native the attributes are `dataComponent`, `dataElement`, and `dataSourceFile`.
+For React on the web the attributes are `data-component`, `data-element`, and `data-source-file`. For React Native the attributes are `dataComponent`, `dataElement`, and `dataSourceFile`, or [`fsTagName`](https://developer.fullstory.com/mobile/react-native/auto-capture/set-tag-name/) and `dataSourceFile` with `setFSTagName` enabled.
```

Comment:
This change makes it really confusing. I recommend an expanded explanation in the React Native section below. 

`fsTagName` is set in the same way as `fsAttribute` and `fsClass`. To mention `fsTagName` here and mentioning `dataSourceFile` twice makes it really confusing.

Reply:
Maybe 
```suggestion
For React on the web the attributes are `data-component`, `data-element`, and `data-source-file`. For React Native the attributes are `fsTagName` and `dataSourceFile`.

(In backwards compatibility mode, the React Native attributes are `dataComponent`, `dataElement`, and `dataSourceFile`.)
```

And then below we could describe omitting `fsTagName` as "backwards compatibility mode". What do you think?

Reply:
The section right below this has a web example of how this plugin will change the JS output. In my opinion, keeping the copy the same will be less confusing. 

The reason is, without our new `setFSTagName`, native rewrites behave very similarly to web, and the web example gives readers a good idea on how this plugin works, because in their head they are thinking `dataComponent`, `dataElement`, and `dataSourceFile` maps to the web attributes.

When we introduce `fsTagName` as an attribute, and introduce the concept of "backward compatibility mode", in my opinion that's very confusing, and React Native users won't see the web example as an accurate portrayal of what's happening in the plugin.

I believe keeping this description the same, and introducing `setFSTagName` in its own section below all this, will avoid this confusion. Users will understand what this plugin is doing without `setFSTagName`. And once they read about `setFSTagName` they will better understand what is changing with this new setting.